corehq/apps/style/static/dashboard/less/dashboard.less

```diff
@@ -52,6 +52,7 @@
   .panel-heading {
     font-weight: bold;
     font-size: @font-size-large;
+    text-align: center;
```

Comment:
How hard would it be to use a PredictablyRandom toggle for this and then
correlate the kissmetrics data with it?